alembic/versions/d05669b60ebe_migrate_agents_to_orm.py

```diff
@@ -0,0 +1,164 @@
+"""Migrate agents to orm
+
+Revision ID: d05669b60ebe
+Revises: c5d964280dff
+Create Date: 2024-12-12 10:25:31.825635
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "d05669b60ebe"
+down_revision: Union[str, None] = "c5d964280dff"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
```

Comment:
why did we replace the table? just wondering

Reply:
OH just to keep naming convention with the other relationship tables, and I figured no one is really using sources (yet) as it's severely broken. original table def was in `metadata.py` called `agent_source_mapping`